misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/model/JClass.java

```diff
@@ -39,68 +38,66 @@
 
 /**
  * Class for storing details pertaining to a specific Java class used for Ballerina bridge code generation.
+ *
+ * @since 1.2.0
  */
 public class JClass {
 
-    public String shortClassName;
-    public String packageName;
-
     private String prefix;
     private String className;
+    private String packageName;
+    private String shortClassName;
 
-    private Boolean isInterface = false;
-    private Boolean directClass = false;
-    private Boolean isAbstractClass = false;
-    private Boolean singleConstructor = false;
-    private Boolean singleConstructorError = false;
-
-    private Set<Class> superClassObjects = new HashSet<>();
+    private boolean isInterface = false;
+    private boolean isDirectClass = false;
+    private boolean isAbstract = false;
```

Comment:
Assigned variables have never accessed?

Reply:
This field is used in the handlebars template.